### PR TITLE
SAK-48856: Error in assignment list view

### PR DESCRIPTION
**Pull request by victorGomollon**
Asociate to [SAK-48856: Error in assignment list view](https://sakaiproject.atlassian.net/browse/SAK-48856)

### CODE REVIEW

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java [hunk 1]
```diff
@@ -5075,4 +5075,14 @@ public String intToTime(int time) {
     public String getContentReviewServiceName() {
         return this.contentReviewService.getServiceName();
     }
+    
+    @Override
+    public String getUserAssignmentModify(String modifier) {
+    	try {
+    		return userDirectoryService.getUser(modifier).getDisplayName();
+    	}catch (UserNotDefinedException e) {
+    		return resourceLoader.getString("user.modify.unknown", "");
```
Comment (bgarciaentornos) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java:5084]:
```suggestion
    		return resourceLoader.getString("user.modify.unknown");
```

Is the extra param necessary?

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java [hunk 2]
```diff
@@ -5075,4 +5075,14 @@ public String intToTime(int time) {
     public String getContentReviewServiceName() {
         return this.contentReviewService.getServiceName();
     }
+    
+    @Override
+    public String getUserAssignmentModify(String modifier) {
+    	try {
+    		return userDirectoryService.getUser(modifier).getDisplayName();
+    	}catch (UserNotDefinedException e) {
```
Comment (bgarciaentornos) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java:5083]:
```suggestion
    	} catch (UserNotDefinedException e) {
```

Comment (bgarciaentornos) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java:5083]:
```suggestion
    @Override
    public String getUserAssignmentModify(String modifier) {
        try {
            return userDirectoryService.getUser(modifier).getDisplayName();
        } catch (UserNotDefinedException e) {
```

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java (hunk 1 above)
Comment (victorGomollon) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java:5084]: yes, if the properties do not exist, it takes the default value ""

File: assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java [hunk 3]
```diff
@@ -878,4 +878,7 @@ public String getDeepLinkWithPermissions(String context, String assignmentId, bo
      * @return A String containing the name of the content review service
      */
     public String getContentReviewServiceName();
+    
+    public String getUserAssignmentModify(String modifier);
```
Comment (mpellicer) [inline assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java:882]: getUserAssignmentModify sounds weird to me, maybe 'getUserWhoModifiedAssignment' ?

**Pull request merged**